apps/ideaspace/src/components/modules/EditIdea/EditIdea.js

```diff
@@ -68,37 +67,42 @@ function EditIdea() {
     status: '',
   });
 
-  const rejectAuthor = async () => {
-    if (!(await confirmArchive())) {
-      router.push(`/ideaspace/workshop/${ideaId}`);
+  const [getError, setGetError] = React.useState(false);
+  React.useEffect(() => {
+    const rejectUser = async () => {
+      if (!(await confirmNotAuthor())) {
+        window.history.back(-1);
+      }
     }
-  };
 
-  const rejectUser = async () => {
-    if (!(await confirmNotAuthor())) {
-      window.history.back(-1);
+    const rejectAuthor = async () => {
+      if (!(await confirmArchive())) {
+        router.push(`/ideaspace/workshop/${ideaId}`);
+      }
     }
-  };
 
-  useEffect(async () => {
     if (ideaId) {
-      const idea = cleanData(await agent.Ideas.getIdea(ideaId, new URLSearchParams("populate=*")));
-      if (!idea || !idea.id || idea.id == 0) {
-        setGetError(true);
-        return;
-      }
+      axios.get(`${process.env.NEXT_PUBLIC_STRAPI_URL}/idea-cards/${ideaId}`)
```

Comment:
Same request as above... is there any reason we are removing references to the agent?